fastify.js

```diff
@@ -15,7 +16,9 @@ function build (options) {
   if (typeof options !== 'object') {
     throw new TypeError('Options must be an object')
   }
+  if (options.logger === true) options.logger = {}
 
+  const logger = pinoHttp(options.logger)
```

Comment:
we need to handle the destination stream, as the second parameter of `pino-http`, see https://github.com/pinojs/pino-http#pinohttpoptions-stream. This might be _really_ awkward, so feel free to go ahead and send a PR against pino-http to get an option for the destination stream.